indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -70,6 +70,15 @@ class BookingDetails extends React.Component {
   state = {
     occurrencesVisible: false,
     activeConfirmation: null,
+    activeAcceptanceForm: false,
```

Comment:
`acceptanceFormVisible`? (this one is a boolean while `activeConfirmation` above which you probably used as a naming hint is a string indicating *which* confirm prompt is visible)